code/src/zarr_converter/zarr_converter_params.py

```diff
@@ -30,13 +30,30 @@ class OmeZarrParams(DefaultSchema):
         # dump_default=[2, 2, 2]
     )
     
+    physical_pixels = List(
+        Float(),
+        required=True,
+        metadata={
+            'description':'Physical pixel sizes in microns in ZYX order'
+        },
+        cli_as_single_argument=True,
+        dump_default=[2.0, 1.8, 1.8]
+    )
+    
     pyramid_levels = Int(
         required=False,
         metadata={
             'description':'number of pyramid levels for ome-zarr multiscale'
         },
         dump_default=5
     )
+    

```

Comment:
Maybe you can do validation here?  It's been awhile, but I think you can set up validation rules in a subclass.